en/gps_compass/README.md

```diff
@@ -40,7 +40,7 @@ Device | GPS | Compass | [RTK](../gps_compass/rtk_gps.md) | [GPS Yaw Output](#co
 [CubePilot Here3 CAN GNSS GPS (M8N)](https://www.cubepilot.org/#/here/here3) | M8P | ICM20948 | &check; | | |
 [Drotek DP0804](https://store-drotek.com/920-DP0804.html) (and other [Drotek u-blox GPS/Compasses](https://store-drotek.com/index.php?controller=search&s=ublox+compass)) | M9N | LIS3MDL | | | |
 [Drotek SIRIUS RTK GNSS ROVER (F9P)](https://store-drotek.com/911-sirius-rtk-gnss-rover-f9p.html) | F9P | RM3100 | &check; | | &check; |
-[Emlid Reach M+](https://emlid.com/reach/)  - PX4 only supports "ordinary" GPS with this module. RTK support is expected in the near future. | &check; | &cross; | | | |
+[Emlid Reach M+](https://emlid.com/reach/)  - PX4 only supports "ordinary" GPS with this module. RTK support is expected in the near future. | &check; | &cross; | | | |&check;
```

Comment:
Hi @moreba1 

Thanks 
- this appears to be called M2 now - is that correct?
- Is the stuff about RTK not being supported in PX4 still correct?

```suggestion
[Emlid Reach M2](https://emlid.com/reach/)  - PX4 supports "ordinary" GPS and PPK with this module. RTK support is expected in the near future. | &check; | &cross; | | | |&check;
```

Reply:
> Hi @moreba1
> 
> Thanks
> 
> * this appears to be called M2 now - is that correct?
> * Is the stuff about RTK not being supported in PX4 still correct?

Reach M+ and M2 both support PPK . I don't know about providing RTK Support for Reach M+ Currently.

Reply:
Thanks very much!